tools/distpartitioning/convert_partition.py

```diff
@@ -183,7 +185,7 @@ def create_dgl_object(graph_name, num_parts, \
 
     return compact_g2, node_map_val, edge_map_val, ntypes_map, etypes_map
 
-def create_metadata_json(graph_name, num_nodes, num_edges, num_parts, node_map_val, \
+def create_metadata_json(graph_name, num_nodes, num_edges, part_id, num_parts, node_map_val, \
```

Comment:
i don't understand. you changed the interface of `create_metadata_json` but doesn't change the place where it is called. how does it work?
also, why is this change related to the bug.